src/NuGet.Services.Staging.Runner/Program.cs

```diff
@@ -118,9 +119,9 @@ private static void InitializeConfiguration(string environment)
             var builder = new ConfigurationBuilder()
                 .SetBasePath(Directory.GetCurrentDirectory())
                 .AddJsonFile("appsettings.json")
-                .AddJsonFile(Path.Combine("Config", $"config.{environment}.json"));
+                .AddJsonFile(Path.Combine("Config", $"config.{environment}.json")).Build();
```

Comment:
>Build [](start = 83, length = 5)

please do this in a separate line. The builder variable is for ConfigurationBuilder, and Build returns an ConfigRoot, so the name is wrong